app/src/androidTest/java/ch/hikemate/app/navigation/SignInAndMapNavigationTest.kt

```diff
@@ -0,0 +1,71 @@
+package ch.hikemate.app.navigation
+
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.performClick
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import ch.hikemate.app.HikeMateApp
+import ch.hikemate.app.ui.navigation.Route
+import ch.hikemate.app.ui.navigation.Screen
+import ch.hikemate.app.ui.navigation.TEST_TAG_DRAWER_ITEM_PREFIX
+import ch.hikemate.app.ui.navigation.TEST_TAG_SIDEBAR_BUTTON
+import ch.hikemate.app.ui.theme.HikeMateTheme
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class SignInAndMapNavigationTest {
+  // Set up the Compose test rule
+  @get:Rule val composeTestRule = createComposeRule()
+
+  @Test
+  fun testInitialScreenIsAuthScreen() {
+    composeTestRule.setContent { HikeMateTheme { HikeMateApp() } }
+
+    composeTestRule.onNodeWithTag(Screen.AUTH).assertIsDisplayed()
+  }
+
+  @Test
+  fun testNavigationToMapScreen() {
+    composeTestRule.setContent { HikeMateApp() }
+
+    composeTestRule.onNodeWithTag(Screen.AUTH).assertIsDisplayed()
+
+    composeTestRule.onNodeWithTag("loginButton").performClick()
```

Comment:
Nitpick: Think about using constants here for the test tag. I've seen this done in some other tests by our team and you've also used them in some other tests here, but its also not a necessity at all.